src/functions/frontend/rctToLft.m

```diff
@@ -124,9 +126,15 @@
     end
     
     delta_cell = cell(size(delta_norm,1), 1);
-    
-    for i = 1:size(delta_norm)
-        delta_cell{i} = toDelta(delta_norm{i});
+
```

Comment:
I think your previous approach with `nargin == 1` and `nargin == 2` was better than checking the length of `varargin`.

Reply:
I think there needs to be a consistency check here too. The user could have `udyn` objects in the `rct_obj`, but (erroneously) not provide a `delta_map`.  This function should throw an error explaining that ('Must provide a definition of each `udyn` object to convert to `Ulft`...').